spaces/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Address(models.Model):
+    street_name = models.CharField(max_length=140)
+    civic = models.CharField(max_length=20)
+    floor = models.IntegerField(blank=True)
+    door = models.CharField(max_length=2, blank=True)
+    city = models.CharField(max_length=140)
+    postal_code = models.IntegerField(blank=True)
+    region = models.CharField(max_length=140)
+    nation = models.CharField(max_length=140)
+
+
+class Place(models.Model):
```

Comment:
A Place needs GPS coordinates. We need to be able to put it on a map. At least, we need latitude and longitude to draw a point on a map.